### ISSUE TITLE
Calendar: Two months in the month Picker calendar is having aria-selected property as true when only one is selected by user.

### ISSUE THREAD

**Issue by dzearing** [OfficeDev/office-ui-fabric-react#11141]
Tracking all of the issues brought up with the 03/15 accessibility pass on Calendar fabric control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8cfe67c5-ce76-4083-b60e-4190dbd38206%7D&action=default&uid=%7B8CFE67C5-CE76-4083-B60E-4190DBD38206%7D&ListItemId=1872&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

**Comment by lorejoh12** [comment]
We currently have two different "selected" dates. "Today" is auto-highlighted with dark blue, and the "selected date" (selected by the user) is light blue. I guess semantically we only want the user selected date, but doing so means we're losing out on informing the user about "today" highlighting. I guess we could just toss that information into the aria label for that cell.

**Comment by lorejoh12** [comment]
Upon further inspection, the real bug here is that for month view specifically, we're marking "today" as selected even when it's not highlighted. This has been fixed in date-time, but needs to be fixed in OUFR Calendar as well.

**Comment by lorejoh12** [comment]
For some reason the PR merging didn't auto close the issues... closing, this should be fixed now.
